.github/workflows/cmwl_pipeline_ci.yml

```diff
@@ -0,0 +1,49 @@
+name: CMWL Pipeline CI
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    branches:
+      - develop
+      - master
+
+jobs:
+  lint:
```

Comment:
Why do we need two jobs?
Why don't we do lint check inside build?

I'm asking, because we're doing Java and Scala setup in both jobs, each setup takes ~10 seconds. 
If we do lint check in the build job, we will save these seconds...

Reply:
I wanted to make them separate jobs because it's a quite common mistake to not format code appropriately. It would have been nice to quickly see that your code is badly formatted instead of going to build details to see what went wrong.
Anyway, now it's changed and build is faster than ever.